components/concept/engine/src/main/java/mozilla/components/concept/engine/selection/SelectionActionDelegate.kt

```diff
@@ -36,4 +38,11 @@ interface SelectionActionDelegate {
      * @returns [true] if the action was consumed.
      */
     fun performAction(id: String, selectedText: String): Boolean
+
+    /**
+     * Takes in a list of actions and sorts them.
+     *
+     * @returns the sorted list.
+     */
+    fun sortedActions(actions: Array<String>): Array<String>
```

Comment:
I know this is still not following the comment [here](https://github.com/mozilla-mobile/android-components/pull/7564#discussion_r449081157). Can you help me understand what syntax I need here? Can't figure it out 😓 

@pocmo

Reply:
Looking at the code as it is right now I think it looks good and keeping it as-is works for me. Can you add kdoc everywhere this is used to explain what `String` is in that `Context`? Otherwise it is impossible write a sorter function without looking at the internals. :)